EssentialsDiscord/src/main/java/net/essentialsx/api/v2/events/discord/DiscordChatMessageEvent.java

```diff
@@ -0,0 +1,72 @@
+package net.essentialsx.api.v2.events.discord;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Fired before a chat message is about to be sent to a Discord channel.
+ * Should be used to block chat messages (such as staff channels) from appearing in Discord.
+ */
+public class DiscordChatMessageEvent extends Event implements Cancellable {
```

Comment:
Stupid and probably nitpicky question, but doesn't the fact that this is fired **before** the message is actually send to Discord make it a **Pre**DiscordChatMessageEvent?

Reply:
The fact that it is Cancellable implies this itself. For example, AsyncPlayerChatEvent doesn't have a `Pre` in it even though it's dispatched before the chat message is sent.